www/assets/vendors.js.spt

```diff
@@ -0,0 +1,16 @@
+import os
+from gratipay.utils import http_caching
+asset_root = os.path.join(website.www_root, 'assets')
+[---]
+
+# Load order
+files = [
+    'jquery.min.js',
```

Comment:
It would be good if we could somehow separate JS, CSS and images into different directories though